lib/aarch64-apple-darwin20.jl

```diff
@@ -10632,28 +10760,28 @@ A scheduled function.
 const aws_task_fn = Cvoid
 
 """
-    __JL_Ctag_375
+    union (unnamed at /home/runner/.julia/artifacts/f0ceef6c683b6b7af5fd02481accb32f8bac0486/include/aws/common/task_scheduler.h:40:5)
 
 honor the ABI compat
 """
-struct __JL_Ctag_375
+struct var"union (unnamed at /home/runner/.julia/artifacts/f0ceef6c683b6b7af5fd02481accb32f8bac0486/include/aws/common/task_scheduler.h:40:5)"
```

Comment:
@Octogonapus, I'm not sure it really matters, but this seems like an exceptionally odd struct name. Just pinging in case you know if there's an easy way to avoid.